howfairis/cli.py

```diff
@@ -94,26 +100,53 @@ def cli(url=None, branch=None, config_file=None, remote_config_file=None, path=N
         print("It seems you have not yet added the fair-software.eu badge to " +
               "your {0}. You can do so by pasting the following snippet:\n\n{1}"
               .format(checker.readme.filename, badge))
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(1)
 
     if current_compliance == previous_compliance:
         print("Expected badge is equal to the actual badge. It's all good.\n")
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(0)
 
     if current_compliance > previous_compliance:
         print("Congratulations! The compliance of your repository exceeds " +
               "the current fair-software.eu badge in your " +
               "{0}. You can replace it with the following snippet:\n\n{1}"
               .format(checker.readme.filename, badge))
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(1)
 
     print("The compliance of your repository is different from the current " +
           "fair-software.eu badge in your " +
           "{0}. Please replace it with the following snippet:\n\n{1}"
           .format(checker.readme.filename, badge))
-
+    github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
     sys.exit(1)
 
 
+def github_readme_creation_check(url, filename, platform, branch):
+    if platform != Platform.GITHUB:
+        return(0)
+    if branch is None:
+        branch = "master"
+    try:
+        response = requests.get(url+"/blob/"+branch+"/"+filename)
+        date_string = BeautifulSoup(response.text, "html.parser").select("relative-time")[0]["datetime"]
```

Comment:
I'd prefer to use the API instead of BeautifulSoup

Reply:
Ok, the call to BeautifulSoup has been replaced with a call to the Github API